ci/docker/ruster/Dockerfile

```diff
@@ -32,18 +36,33 @@ RUN set -eux; \
     chmod -R a+w $RUSTUP_HOME $CARGO_HOME; \
     rustup --version; \
     cargo --version; \
-    rustc --version; \
-    apt-get remove -y --auto-remove \
-        wget \
-        ; \
-    rm -rf /var/lib/apt/lists/*;
+    rustc --version
 
-RUN rustup target add \
-        x86_64-unknown-linux-gnu \
-        i686-unknown-linux-gnu \
-        armv7-unknown-linux-gnueabihf \
-        aarch64-unknown-linux-gnu \
-        arm-unknown-linux-gnueabi
+RUN set -eux; \
+  rustup target add \
+    x86_64-unknown-linux-gnu \
+    i686-unknown-linux-gnu \
+    armv7-unknown-linux-gnueabihf \
+    aarch64-unknown-linux-gnu \
+    arm-unknown-linux-gnueabi
+
+RUN set -eux; \
+  mkdir -p /usr/lib/i386-linux-gnu \
+    /usr/lib/x86_64-linux-gnu \
+    /usr/lib/arm-linux-gnueabihf \
+    /usr/lib/arm-linux-gnueabi \
+    /usr/lib/aarch64-linux-gnu
+
+RUN --mount=type=secret,id=gl_access_token export GL_ACCESS_TOKEN="$(cat /run/secrets/gl_access_token)"; \
```

Comment:
doesn't this mean that if the token expires the image cannot be used anymore?

Reply:
The `--mount-type=secret(...)` is used only at build time and the secret is available only for the `RUN` command it's used in. After the command is executed, the secret is unmounted so it's not stored in the image. So this is build-only feature and if the token expires later it doesn't matter because it's not stored in the image and it's not used at runtime